$OpenBSD: patch-concurrent-extra_cabal,v 1.1 2012/12/02 20:46:08 kili Exp $

Builds fine with stm-2.4.
The testsuite passes fine with it (and test-framework-0.6).
Note: To be able to run the testsuite, you have to build with
MODGHC_SETUP_CONF_ARGS = --enable-tests

--- concurrent-extra.cabal.orig	Sun Apr  8 15:33:58 2012
+++ concurrent-extra.cabal	Fri Nov 30 22:30:29 2012
@@ -52,7 +52,7 @@ source-repository head
 library
   build-depends: base                 >= 3       && < 4.6
                , base-unicode-symbols >= 0.1.1   && < 0.3
-               , stm                  >= 2.1.2.1 && < 2.4
+               , stm                  >= 2.1.2.1 && < 2.5
                , unbounded-delays     >= 0.1     && < 0.2
   exposed-modules: Control.Concurrent.Lock
                  , Control.Concurrent.STM.Lock
@@ -82,10 +82,10 @@ test-suite test-concurrent-extra
 
   build-depends: base                 >= 3       && < 4.6
                , base-unicode-symbols >= 0.1.1   && < 0.3
-               , stm                  >= 2.1.2.1 && < 2.4
+               , stm                  >= 2.1.2.1 && < 2.5
                , unbounded-delays     >= 0.1     && < 0.2
                , HUnit                >= 1.2.2   && < 1.3
-               , test-framework       >= 0.2.4   && < 0.5
+               , test-framework       >= 0.2.4   && < 0.7
                , test-framework-hunit >= 0.2.4   && < 0.3
 
 -------------------------------------------------------------------------------
